### PR TITLE
#64 Implemented automatic inspection in case grossAmount < 200.

### PR DESCRIPTION
**Pull request by dzhitomirsky-sc**
* Implemented automatic approval in case invoice.grossAmount < 200
* added unit tests
* added error description for invoice items import

### CODE REVIEW

File: src/server/workflow/ApprovalSchema.spec.js [hunk 1]
```diff
@@ -184,4 +187,16 @@ describe("Invoice approval flow", () => {
       });
     });
   });
+
+  describe('check of aotomatic transitions', () => {
```
Comment (apechenko-sc) [inline src/server/workflow/ApprovalSchema.spec.js]: spelling mistake in word 
> automatic

File: src/server/workflow/conditions/isLessThan.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Checks that invoice.field < <amount>
```
Comment (apechenko-sc) [inline src/server/workflow/conditions/isLessThan.js]: should be object.field probably

File: src/server/workflow/conditions/isLessThan.spec.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const assert = require('assert');
+const isLessThan = require('./isLessThan');
+const expect = require('chai').expect;
+/**
+ * Unit test for isLessThat conditions
```
Comment (apechenko-sc) [inline src/server/workflow/conditions/isLessThan.spec.js]: should be isLessThan

**Comment by dzhitomirsky-sc** [comment]
Fixed misprints.

**Pull request merged**